sql-plugin/src/main/321cdh/scala/com/nvidia/spark/rapids/spark321cdh/RapidsShuffleManager.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020-2022, NVIDIA CORPORATION.
+ * Copyright (c) 2021, NVIDIA CORPORATION.
```

Comment:
revert copyright back to 2020-2022 since moved

Reply:
done